### ISSUE TITLE
Does WilliamChart support touch and drag?

### ISSUE THREAD

**Issue by DSofter** [diogobernardino/williamchart#193]
Like this:
![touch](https://cloud.githubusercontent.com/assets/1687829/24334199/5299bca0-121b-11e7-982f-6fef19940aa5.gif)

Thank you!

**Comment by diogobernardino** [comment]
No, not supported, though interesting if not introducing much code. I might look into it in future.

**Comment by DSofter** [comment]
@diogobernardino Good to know! Love your library! This will be a cool feature to have especially for line charts!@diogobernardino Good to know! Love your library! This will be a cool feature to have especially for line charts!

**Comment by RNYadav** [comment]
@diogobernardino it will be great if you could allow the user to listen to touch events on the chart, to show the exact value and trigger some kind of action. I think drag feature can wait but the touch feature is a must on any kind of graph/chart. I loved the simplicity of the Charts. Great work!!

**Comment by diogobernardino** [comment]
I've been implementing the foundation to support user interactions as the one initially shown. This is the result so far, feeling optimistic about it 👇

![cropped](https://user-images.githubusercontent.com/1091075/84174770-5319f500-aa7f-11ea-9fba-d530bd536262.gif)

@RNYadav Click listener is already part of `3.5.0` and supported by all charts except for `DonutChartView`. Touch listener is coming soon. I intend to keep them as experimental features for the time being.

**Comment by diogobernardino** [comment]
Tooltips support is now available from version `3.7.0`. The sliding bar behaviour is a separate tooltip dependency:

```gradle
implementation 'com.diogobernardino.williamchart:tooltip-slider:3.7.0'
```
